enhancements/two-nodes-openshift/2no.md

```diff
@@ -91,17 +97,17 @@ This requires our solution to provide a management experience consistent with "n
 * Support disconnected cluster installation
 * Adding worker nodes
 * Creation of RHEL-HA events and metrics for consumption by the OpenShift monitoring stack (Deferred to post-MVP)
-* Supporting upgrade/downgrade paths between two-node and other architectures (for initial release)
+* Supporting upgrade/downgrade paths between two-node and other topologies (e.g. 3-node compact) (for initial release)
 
 ## Proposal
 
-Use the RHEL-HA stack (Corosync, and Pacemaker), which has been used to deliver supported 2-node cluster experiences for multiple decades, to manage cri-o, kubelet, and the etcd daemon.
-etcd will run as as a voting member on both nodes.
+We will use the RHEL-HA stack (Corosync, and Pacemaker), which has been used to deliver supported 2-node cluster experiences for multiple decades, to manage cri-o, kubelet, and the etcd daemon.
+Etcd will run as a a voting member on both nodes.
```

Comment:
etcd is correct https://etcd.io/

Reply:
I will rephrase this sentence so it doesn't conflict with english grammar rules :)